### ISSUE TITLE
[Bug]: Colors and Fonts dialog does not remember size/pos

### ISSUE THREAD

**Issue by uffejakobsen** [eranif/codelite#3127]
### What happened?

[Bug]: Colors and Fonts dialog does not remember size/pos

Seen on latest master

### Version

Self compiled

### Operating system

Linux

**Comment by eranif** [comment]
Yes, I am aware of this. This is because of the new `wxCollapsiblePane`. It does not play nice with size of dialog as it adjust the dialog size once its clicked...
Click it and you will notice it. Remembering the size while expanded and then re-opening the dialog will make it huge...
